aiven/client/cli.py

```diff
@@ -5626,6 +5631,83 @@ def organization__user__invite(self) -> None:
         """Invite user to join an organization"""
         self.client.invite_organization_user(self.args.organization_id, self.args.email)
 
+    @arg.json
+    @arg.organization_id
+    def organization__group__list(self) -> None:
+        """List user groups in an organization"""
+        groups = self.client.list_user_groups(self.args.organization_id)
+        layout = [
+            "user_group_name",
+            "description",
+            "user_group_id",
+            "member_count",
```

Comment:
looks so close `USER_GROUP_COLUMS` almost feels like it could `USER_GROUP_COLUMNS + ['member_count']`  but that also feels like trying to be too DRY 🤷 

But at least same argument for ordering of colums applies.

Reply:
Not anymore, took description to the very last